bundles/org.openhab.binding.lgthinq/src/main/java/org/openhab/binding/lgthinq/internal/LGThinqHandlerFactory.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lgthinq.internal;
+
+import static org.openhab.binding.lgthinq.internal.LGAirConditionerHandler.THING_TYPE_AIR_CONDITIONER;
+import static org.openhab.binding.lgthinq.internal.handler.LGBridgeHandler.THING_TYPE_BRIDGE;
+
```

Comment:
THING_TYPEs have to be defined in LGThinqBindingConstants ad be imported from there.
Furthermore, naming is not correct, it should be LGThinqBridgeHandler and LGThinqAirConditionerHandler